ci/dev/vscode.patch

```diff
@@ -922,8 +924,57 @@ index 0000000000000000000000000000000000000000..3c0703b7174ad792a4b42841e96ee937
 +		});
 +	}
 +
++	const logService = (services.get(ILogService) as ILogService);
++	const storageService = (services.get(IStorageService) as IStorageService);
++	// We set this here first in case the path changes.
++	const updateCheckEndpoint = path.join(window.location.pathname, "/update/check")
```

Comment:
I think we'll want to use `options.base` at some point to make this independent of the vscode endpoint (for example if you access it at `/vscode` instead of `/`).

Reply:
There is options.base anymore. It was removed.

Reply:
I'll add it back though.